test/config-luatest/log_wrapper_test.lua

```diff
@@ -6,23 +6,6 @@ local justrun = require('luatest.justrun')
 
 local g = t.group()
 
-g.test_jsonify_table = function()
```

Comment:
I don't see any point in removing tests in a separate commit. We usually add/remove tests in the same commit that adds/removes functionality.

Reply:
No problems, will do at the next push.

Reply:
Done.